kbi/0020/index.rst

```diff
@@ -0,0 +1,35 @@
+.. index::
+   single: datalad;installation
+
+KBI0020: Installing DataLad and dependencies in Google Colab
+============================================================
+
+:authors: Stephan Heunis <jsheunis@gmail.com>
+:discussion: <link>
+:keywords: workaround, credentials, interactive authentication prompt, keyring
```

Comment:
Looks like they got copied from a previous KBI :stuck_out_tongue_winking_eye: feel free to adjust.
```suggestion
:keywords: Google Colab, Conda, installation```